models/BorderOptions/BorderOptions.js

```diff
@@ -31,6 +31,7 @@ class BorderOptions {
         dashPattern: args.dashPattern,
         lineCapStyle: BorderOptions.LineCapStyle[args.lineCapStyle || 'butt'],
         lineJoinStyle: BorderOptions.LineJoinStyle[args.lineJoinStyle || 'miter'],
+        isEnabled: args.isEnabled || true,
```

Comment:
We might need to change this, if the user supplies `false` then the first part of this conditional will be false but the second part is true so it will return true. Take a look at this and let me know what you think. Not the most readable/understandable, but `false` and `true` would be set properly and `undefined` would return `true`:

https://github.com/amzn/sketch-constructor/blob/master/models/Fill/Fill.js#L41

Reply:
Good point. What do you think about ` isEnabled = !(args.isEnabled === false),`

Reply:
Correct, `args.isEnabled !== false` would return true for undefined. So I guess depending on what we want the default value to be. I believe both snippets in your comment would have the same results, so I guess whichever one makes more sense.

Reply:
@dbanksdesign just pushed a fix. hope we can merge it now :)